<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_sever">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>접속</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_push">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 업로드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pull">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 받기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textv">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>191</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>현제 디렉토리</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>122</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>서버 디렉토리</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>상태창</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listw">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_ed">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_pw">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>54</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PassWord</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>실험작</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_ip">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_sto">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>53</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Host_IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>85</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_pt">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_id">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>ID 기억하기</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_sto">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Storage 기억하기</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_ip">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>IP,Port 기억하기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>114</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Storage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_f">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>찾기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
